examples/lighting-app/silabs/efr32/BUILD.gn

```diff
@@ -56,6 +56,12 @@ efr32_sdk("sdk") {
     "${examples_common_plat_dir}",
   ]
 
+  if (use_rs911x) {
```

Comment:
This shouldn't go there. @jmartinez-silabs did a cleanup of all of the GN files. It should go in the comment platform.

Reply:
Okay @jepenven-silabs we don't know about the @jmartinez-silabs PR we moved this to efr32_sdk.gni